docs/source/development/coding_conventions.rst

```diff
@@ -231,6 +231,56 @@ corresponds to complexity of the body).
 
 |x| **Do not** declare imports not used within the file.
 
+Element ID
+----------
```

Comment:
I think it makes sense to add a **short** motivation for the id naming convention. This is a little different than what the introduction of this document covers, so I've got no problem with that.